keyboards/skyloong/gk61/pro/config.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 linlin012 (@linlin012)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+
+#ifdef RGB_MATRIX_ENABLE
```

Comment:
Sorry, this wasn't meant to be removed in my review.
```suggestion
#ifdef RGB_MATRIX_ENABLE
#define DRIVER_COUNT 1
```